x-pack/plugins/aiops/public/components/change_point_detection/change_point_detection_root.tsx

```diff
@@ -51,25 +54,42 @@ export const ChangePointDetectionAppState: FC<ChangePointDetectionAppStateProps>
 
   const warning = timeSeriesDataViewWarning(dataView, 'change_point_detection');
 
+  const reload$ = useMemo<Observable<number>>(() => {
+    return mlTimefilterRefresh$.pipe(map((v) => v.lastRefresh));
+  }, []);
+
   if (warning !== null) {
     return <>{warning}</>;
   }
 
+  const PresentationContextProvider = appDependencies.presentationUtil!.ContextProvider;
```

Comment:
Even though presentationUtil is a required plugin, is there a possibility that this still could be undefined?

Reply:
Updated in https://github.com/elastic/kibana/pull/162796/commits/5d0bbf30e4f669227115c0d4beba05d1b76024fd